spec/domain/healthchecks/monthly_aggregations_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe Healthchecks::MonthlyAggregations do
+  include_examples 'Healthcheck enabled/disabled within time range'
+
+  around do |example|
+    Timecop.freeze(Date.new(2019, 2, 22)) { example.run }
+  end
+
+  its(:name) { is_expected.to eq(:aggregations) }
+
+  describe '#status' do
+    context 'When there are aggregations' do
+      before do
+        date = Date.yesterday
+        edition = create :edition, date: date
```

Comment:
There is no need to set `date` for the `Edition`, we only need it for the metric.

Reply:
Thanks - addressed in https://github.com/alphagov/content-performance-manager/pull/1153/commits/6a00f3cdfb70e20e266eb93af9c07911eb28a6da